pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -198,6 +199,11 @@ func (r *ReconcileSyncSetInstance) Reconcile(request reconcile.Request) (reconci
 		return reconcile.Result{}, err
 	}
 
+	if cd.Annotations != nil && cd.Annotations[constants.AnnotationSyncsetPause] == "true" {
+		ssiLog.Debug("syncing to cluster is disabled")
```

Comment:
Warn here as well, I'd throw in "due to %s annotation" as well in both messages.